emukit/quadrature/kernels/quadrature_kernels.py

```diff
@@ -20,18 +21,36 @@ class QuadratureKernel:
     An example of a specific QuadratureKernel and IStandardKernel pair is QuadratureRBF and IRBF.
     """
 
-    def __init__(self, kern: IStandardKernel, integral_bounds: List[Tuple[float, float]], integral_name: str='') \
+    def __init__(self, kern: IStandardKernel, integral_bounds: Union[None, List[Tuple[float, float]]],
+                 measure: Union[None, IntegrationMeasure], variable_names: str= '') \
             -> None:
         """
         :param kern: standard emukit kernel
         :param integral_bounds: defines the domain of the integral. List of D tuples, where D is the dimensionality
         of the integral and the tuples contain the lower and upper bounds of the integral
-        i.e., [(lb_1, ub_1), (lb_2, ub_2), ..., (lb_D, ub_D)]
-        :param integral_name: the (variable) name(s) of the integral
-        """
+        i.e., [(lb_1, ub_1), (lb_2, ub_2), ..., (lb_D, ub_D)]. None if bounds are infinite
+        :param measure: the integration measure. Note that the integral in the QuadratureKernel are specific to the
+        measure. None if no measure.
+        :param variable_names: the (variable) name(s) of the integral
+        """
+
+        # we define reasonable box bounds for the integral because e.g., the optimizer of the acquisition function
+        # requires finite bounds. The box is defined by the integration measure. See each integration measure for
+        # details. Note that this only affects methods that use this box, e.g. the acqusition optimizers. The integrals
+        # in this kernel will have infinite bounds still.
+        if (integral_bounds is None) and (measure is None):
+            raise ValueError('integral_bounds and measure are both None. At least one of them must be given.')
+        if integral_bounds is None:
+            reasonable_box_bounds = measure.get_box()
```

Comment:
Did you mean to use this value to initialize `self.integral_bounds`?

Reply:
no, the integral bounds are infinity here. However, the optimizer required finite bounds, so I put a reasonable box only for the optimizer. For now, reasonable boxes are the ones outside of which the integration measure is, or virtually is, zero. The integrals are still computed with infinite bounds.

Reply:
oh, i see now, thanks